app/controllers/katello/concerns/organizations_controller_extensions.rb

```diff
@@ -47,6 +48,7 @@ def create
       def update
         return if params[:simple_content_access].nil?
         sca_param = ::Foreman::Cast.to_bool(params[:simple_content_access])
+        Rails.logger.warn "DEPRECATION WARNING: Simple Content Access will be required for all organizations in Katello 4.12."
```

Comment:
Can we use `::Foreman::Deprecation.api_deprecation_warning` here?

Reply:
For both `update` and `create`?